test/unit-test/test_btc_sign.c

```diff
@@ -381,6 +383,15 @@ static void _sign(const _modification_t* mod)
     if (mod->overflow_output_ours) {
         outputs[4].value = ULLONG_MAX;
     }
+    if (mod->litecoin_rbf_disabled) {
+        init_req.coin = BTCCoin_LTC;
+        init_req.locktime = 1;
+        inputs[0].sequence = 0xffffffff - 2;
+        inputs[0].keypath[1] += 2;
```

Comment:
slight preference for `= 0 + BIP32_INITIAL_HARDENED_CHILD`, as `+=` means this value actually depends on `inputs[0].keypath[1]` (even if it changes), but it doesn't.